share/doc/homebrew/Installation.md

```diff
@@ -29,7 +29,7 @@ not build when installed elsewhere. One of the reasons Homebrew just
 works relative to the competition is **because** we recommend installing
 to `/usr/local`. *Pick another prefix at your peril!*
 
-`mkdir homebrew && curl -L https://github.com/Homebrew/homebrew/tarball/master | tar xz --strip 1 -C homebrew`
+`mkdir homebrew && curl -L https://github.com/Homebrew/brew/tarball/master | tar xz --strip 1 -C brew`
```

Comment:
The `-C brew` needs to be changed back to `-C homebrew` as it refers to the directory created via `mkdir` (or the `mkdir` adjusted otherwise).